rust/basic_bitcoin/src/basic_bitcoin/src/types.rs

```diff
@@ -0,0 +1,62 @@
+use ic_cdk::export::{
+    candid::{CandidType, Deserialize},
+    serde::Serialize,
+    Principal,
+};
+
+#[derive(CandidType, Deserialize)]
+pub struct SendRequest {
+    pub destination_address: String,
+    pub amount_in_satoshi: u64,
```

Comment:
See above. In fact, the whole reason I defined `SendRequest` was for clarity in the Candid UI. Otherwise, I wouldn't have defined a `struct` at all and instead passed in both arguments directly to `send`.